svthalia/concrexit
# Add Nix OS-based review environment

### Summary

Adds back the review environment, now based on terraform, nixos and github environments

### How to test
Steps to test the changes you made:
1. Go to a PR
2. Allow the deployment to continue
3. See the review environment on technicie.nl
